ext/ffi/tests/013.phpt

```diff
@@ -57,8 +57,8 @@ try {
 int(1)
 int(2)
 int(3)
-FFI\ParserException: 'void' type is not allowed at line 1
-FFI\ParserException: 'void' type is not allowed at line 1
+FFI\ParserException: Type void is not allowed at line 1
+FFI\ParserException: Type void is not allowed at line 1
```

Comment:
I like the previous message more.

Reply:
The apostrophe bothers me (just a little bit) because we don't quote types anywhere. What about `void type is not allowed?`